### PR TITLE
Serve a copy of the codeprojects footer on dashboard

### PR DESCRIPTION
**Pull request by bethanyaconnor**
Adds codeprojects.org/style.css and codeprojects.org/scripts/hosted.js paths to dashboard. These are actually redirects to paths that make a bit more sense from a code-organization perspective.

The three files/paths added are:
- `dashboard/public/weblab/footer.js` was created by copying `pegasus/sites.v3/codeprojects.org/public/scripts/hosted.js`, applying prettier, and changing the api path called.
- `dashboard/app/views/projects/_weblab_footer.html.haml` was created by copying `pegasus/sites.v3/codeprojects.org/views/footer.haml` and localizing it. It needed to be in the `dashboard/app/views` directory to take advantage of haml compilation, so it's got a lightweight controller action around it
- `dashboard/app/assets/stylesheets/weblab/footer.scss` was created by copying `pegasus/sites.v3/codeprojects.org/styles/050-theme.css` and changing it to sass. The other stylesheet in that directory is just the gotham font, which is the same font dashboard uses. The colors were tweaked very slightly to align with the colors in `shared/css/color.scss`. Also, this file is in a subdirectory of `assets/stylesheets` so that it isn't included by default. I can't quite figure out why this works but without this it breaks the footer for the entire site because it adds padding to `body`.

I tested these changes locally and they work for me (the footer actually looks slightly nicer). When codeprojects.org traffic is directed to dashboard, we should be able to confirm that these are working. Ideally, we will eventually be able to change the paths [here](https://github.com/code-dot-org/code-dot-org/blob/8328c101fdbd4d5c6eb9f82db64f85c98cda5e82/shared/middleware/files_api.rb#L288) to remove the redirects after we move codeprojects off of sinatra.

Screenshot of footer locally:
Screen Shot 2022-06-07 at 3 45 08 PM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/config/routes.rb [hunk 1]
```diff
@@ -216,6 +216,10 @@
   put '/featured_projects/:project_id/unfeature', to: 'featured_projects#unfeature'
   put '/featured_projects/:project_id/feature', to: 'featured_projects#feature'
 
+  get '/weblab/footer', to: 'projects#weblab_footer'
```
Comment (jamescodeorg) [inline dashboard/config/routes.rb]: Could we add a comment here to indicate that these routes are used for weblab shares on codeprojects.org?

**Review by jamescodeorg — APPROVED** [review]
Looks great, thanks for going through all of the little details on this.

There's a small typo in the PR description (codeprojects.com should be codeprojects.org).  Really appreciate that you took the time to write up how these various files were derived!

File: dashboard/config/routes.rb (hunk 1 above)
Comment (bethanyaconnor) [reply dashboard/config/routes.rb]: Sure! I should probably also limit this route to codeprojects like I did with the other routes

**Comment by bethanyaconnor** [comment]
Unit tests passed in https://drone.cdn-code.org/code-dot-org/code-dot-org/19812
UI tests passed in https://drone.cdn-code.org/code-dot-org/code-dot-org/19786

**Pull request merged**